test/endtoend/operator/101_initial_cluster_unmanaged_tablet.yaml

```diff
@@ -8,12 +8,12 @@ metadata:
   name: example
 spec:
   images:
-    vtctld: vitess/lite:latest
-    vtadmin: vitess/vtadmin:latest
-    vtgate: vitess/lite:latest
-    vttablet: vitess/lite:latest
-    vtbackup: vitess/lite:latest
-    vtorc: vitess/lite:latest
```

Comment:
FYI, this is something we missed during a previous backport, should have been changed in the backport to be `v20.0.0`.